lib/scriptLoader.js

```diff
@@ -17,6 +17,9 @@
 
   function webWorkerImport(loader, load) {
     return new Promise(function(resolve, reject) {
+      if (load.metadata.integrity)
+        reject('cannot verify integrity of ' + load.address);
```

Comment:
Actually, never mind about moving it up - would mean doing environment checks too which isn't ideal. Just update the message and we should be good.